igvm/drbd.py

```diff
@@ -7,6 +7,10 @@
 from io import BytesIO
 from logging import getLogger
 from time import sleep
+from typing import Tuple, Optional
+
+from tqdm import tqdm
```

Comment:
Will it handle non-terminal runs, like redirections to a file?

Reply:
well yeah. it will print `\r` in the file though as it would on the terminal. is this a problem for you? will look ugly in `less` for example but print beautifully on your terminal with `cat` or `tail`

Reply:
> 
> disable : bool, optional
>
>   Whether to disable the entire progressbar wrapper [default: False]. If set to None, disable on non-TTY.

Let's check setting it to `None`.

Reply:
works, set to `None`